test/e2e/SingleVoteGovernorE2E.t.sol

```diff
@@ -105,22 +105,15 @@ contract SingleVoteGovernorLifecycle is E2eTest {
 
         authorizer.renounceRole(ownerRole, address(this));
 
-        // Funders deposit funds
-
-        // IMPORTANT
-        // =========
-        // Due to how the underlying rebase mechanism works, it is necessary
-        // to always have some amount of tokens in the orchestrator.
-        // It's best, if the owner deposits them right after deployment.
+        //--------------------------------------------------------------------------------
+        // Set up seed deposit and initial deposit by users
```

Comment:
The initial deposit isnt necessary anymore is it?
I think we should remove the fundingmanager inclusion in this e2e test completely and maybe just focus on the role management side of the singlevote governor
Otherwise it might confuse potential users that the payment part may be strictly necessary although it isnt

You could just do a Role change instead or maybe change something via setter

Reply:
I think the initial deposit is still necessary for the rebasingFundingManger in general, but since in this case we aren't paying out anything it doesn't matter

Reply:
BTW it wasn't, I removed it